.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "tabWidth": 2,
+  "singleQuote": true,
+  "semi": true,
+  "trailingComma": "all",
+  "bracketSameLine": true
```

Comment:
This is deprecated https://prettier.io/docs/en/options.html#deprecated-jsx-brackets

See https://prettier.io/docs/en/options.html#bracket-line